README.md

```diff
@@ -1,2 +1,74 @@
-# Hello
-A simple project to familiarize with tools and best practices in software development
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 MICROVERSE HELLO PROJECT <a name="about-project"></a>
+
+**[microverse hello project]** is a simple project to familiarize with tools and best practices in software development 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project is built with HTML and CSS only
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Demonstrate how to add html markup]**
+- **[Demonstrate adding styles to html markup]**
+- **[Demonstrate linking CSS and HTML files]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+1. Clone this repository
+2. Open it in your preffered text editor
+3. Serve the index.html file in a local web server
+
+### Prerequisites
+
+In order to run this project you need:
+-A web server
+-A browser
+
```

Comment:
- [ ] Kindly make sure you always list all the tools and elements requirements in the readme so that even non-tech folks can use it to learn how to handle your repo.
I think one would need aswell
`NodeJS`
`The version control Git`
`an Code Editor(VSCode...)`
Kindly add that